ampersand-select-view.js

```diff
@@ -291,6 +319,18 @@ SelectView.prototype.setMessage = function (message) {
     }
 };
 
+SelectView.prototype.clearMessage = function () {
+    var mContainer = this.el.querySelector('[data-hook~=message-container]');
+    var mText = this.el.querySelector('[data-hook~=message-text]');
```

Comment:
thoughts to defining these in `render()` vs here?  perf v. mem hit, but it could be nice to expose them?